corehq/apps/app_manager/tests/data/suite/call-center.json

```diff
@@ -119,6 +119,13 @@
                             "doc_type": "SortElement",
                             "field": "top_of_tech_support_list",
                             "type": "plain"
+                        },
+                        {
+                            "direction": "descending",
+                            "doc_type": "SortElement",
+                            "field": "phone_number",
+                            "type": "integer",
+                            "display": { "en": "PhoneNumber" }
                         }
```

Comment:
I just noticed that this is modifying an unrelated test. I think it would be better to update the `test_sort_only_value_suite` test: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/test_suite.py#L55